TASK: Remove close button for relogin dialog

maya-shines opened the pull request:
Former behavior:
When the backend session expires, the relogin dialog forces the user to login again.
This relogin dialog has an "x"-Button to close it, but the button does nothing.

![image](https://user-images.githubusercontent.com/6552092/57841546-08ba3c00-77cb-11e9-9421-dba145b04e70.png)

New behavior:
The relogin dialog still opens, but now no longer has the useless "x"-Button:

![image](https://user-images.githubusercontent.com/6552092/57841598-2687a100-77cb-11e9-98bb-0fd5b93a85f3.png)

Sidenote: The dialog.tsx already checks, if the "onRequestClose" is set to something. However, "{emptyFn}" seems to evaluate as not empty, thus rendering the button, despite it not having a function. All I did was to replace the empty function with an empty string.

danielkestler commented:
Awesome, thanks. While you're over it, would you mind adding some margin after the input fields?

maya-shines commented:
On it!

danielkestler commented:
A while ago, we discussed getting rid of the close button altogether – so if this is easier, maybe we can just remove it from all dialogs?

daniellienert commented:
👍🏼

markusguenther commented:
Marked it as work in progress. Maya told me that she plans additional work on it.

maya-shines commented:
Hey you guys, I added some changes. The close button is now removed from all modals. I could only find one modal that doesn't have a cancel button to act as an exit strategy (other than pressing the esc key or clicking elsewhere). That was the keyboard shortcut modal. So I added a little close button there. Last change was adding some margin to the input fields in the relogin dialog:
![image](https://user-images.githubusercontent.com/6552092/58032075-21f01f00-7b22-11e9-98a3-58d5a05e6093.png)


I hope this will do!

danielkestler commented on packages/neos-ui/src/Containers/Modals/ReloginDialog/style.css:
```diff
@@ -1,7 +1,7 @@
 }
-.inputField {
+.inputFieldWrapper {
     margin-bottom: 15px;
```
Maybe use spacingHalf constant here? Just a nitpick...

maya-shines replied:
done ;)

Sebobo commented:
@maya-shines cool thanks! 
So it's no WIP anymore?

markusguenther commented:
I put the WIP on the PR after Maya mentioned that she wants to continue this tasks after the sprint.
